docs/hardware/hardware-selection.md

```diff
@@ -0,0 +1,278 @@
+
+    Some handsets with built in ELRS include:
+
+      * RadioMaster TX16S ELRS
+      * RadioMaster Zorro ELRS
+      * BetaFPV Lite Radio 3 Pro ELRS
+      * Jumper T-Pro
+      * Jumper T-Lite v2
+
+    A notable exception to this list is the iFlight Commando, which, while including a TX built into the handset is simply wired as an external module, and put within the casing. It does support an external module and notably has both 868/915MHz and 2.4GHz options available
+
+??? info "Frequency"
+
+    ExpressLRS offers both 2.4GHz and 900MHz systems, with each only working with receivers and transmitters of the same frequency. 2.4GHz is the currently most popular frequency given it's legality, featureset and cost. 2.4GHz links offer the fastest packet rates allowing for a more locked in feel while piloting and still offer massive amounts of range. 900MHz is the original long range frequency and still can be great for high wifi pollution environments with slightly better penetration.
+
+    For new users unless you plan on flying hundreds of kilometers or in a high noise environment we reccomend 2.4GHz hardware:
+
+      * Axisflying Thor
+      * RadioMaster Ranger
+      * HappyModel ES24TX Pro
+      * Namimno Flash
+
+    If you want to push the limits of range, a 900MHz system might fit within your needs. Some off the shelf 900MHz transmitters include:
+
+      * Namimno Voyager
+      * Happymodel ES900TX
+      * BetaFPV Micro 915/868MHz
+
+??? info "Size"
+
+    Most ExpressLRS Transmitter modules fit into one of two classes - Micro and Nano. There are some exceptions that fit into multiple or have interesting features that may make them work better for your needs.
+    
+    Micro modules will fit into the JR bay of a standard transmitter such as a TX16s or QX7, with some examples including:
+
+      * RadioMaster Ranger Micro
+      * Namimno Flash
+      * HappyModel ES24TX
+    
+    Nano modules will fit into a lite module bay such as that on a Zorro, T-Pro or X-Lite with some examples including:
+
+      * HappyModel ES24TX Slim Pro
+      * Radimoaster Ranger Nano
+      * Jumper AION Nano
+      * BetaFPV Nano
+
+    Some notable exceptions that can fit into multiple or very specific module bays include:
+
+      * HappyModel ES24TX Lite - Fits onto Jumper T-Lite
+      * Axisflying Thor - Has a wire system to be able to plug into any radio with a CRSF output
+      * Radiomaster Ranger - includes Micro & Nano mounts and wiring for any CRSF compatible radio
+
+??? info "Power"
+
+    Most ELRS Transmitters have a 250mW or 500mW power cap, but for that extra percentage of range, a select few 1W modules exist, packing that punch needed to go even further. These transmitters include large heatsinks, fans and sometimes temperature sensing to cool the power output. Some of these 1W transmitters are:
```

Comment:
This sounds like there's massive range difference between a 500mW and 1W tx modules. Doubling power doesn't always mean double the range/penetration.
consider rewording to "needed to go a bit further".

"temperature sensing to cool the rf components"